pluto/src/icon/Icon.tsx

```diff
@@ -46,8 +46,12 @@ export const Icon = ({
   </div>
 );
 
-export interface CreateProps extends Omit<IconProps, "topRight"> {}
+export interface DetailProps extends Omit<IconProps, "topRight"> {}
 
-export const Create = (props: CreateProps): ReactElement => (
+export const Create = (props: DetailProps): ReactElement => (
```

Comment:
Was this rename intentional?

Reply:
Why is it called `DetailProps`, I don't understand the naming. You could also just create a type alias so you could have `CreateProps` and `ImportProps` and just alias one to the other.